src/css/bundle.scss

```diff
@@ -1139,280 +1139,33 @@ input[type='text'].display {
-    background-size: cover;
-}
-
-body .window-app.forien-quest-preview .quest-body .quest-info .quest-col-right .quest-tasks .toggleState:hover {
-    color: var(--palette-primary,var(--default-primary-accent,$yellow));
-}
-
-body #forien-quest-log .actions i.fa-times-circle, .window-app.forien-quest-preview .actions i.fa-times-circle {
-    color: $red;
-}
-
-body .quest-tasks ul {
-    background: #00000047;
-}
 
-body .window-app.forien-quest-preview .quest-body .quest-info .quest-col-right header {
-    border-block-end: 2px solid var(--palette-primary,var(--default-primary-accent,#782e22));
-    border-bottom: 2px solid var(--palette-primary,var(--default-primary-accent,#782e22));
-    color: $edge;
-    display: flex;
-    flex: 0 0 1px;
-    margin-bottom: 4px;
-    background: #000000b8;
-}
-//table h2
-body #forien-quest-log .quest-log .table ul li.drag-quest {
-    align-items: center;
-    background: #000000b8;
-    border: 1px solid #0000;
-    border-radius: 5px;
-    display: flex;
-    justify-content: flex-start;
-    margin: 0 4px 2px 0;
-    min-height: 42px;
-    transition: border-color .3s ease,box-shadow .3s ease;
-}
-//title h2
-body #forien-quest-log .quest-log .table ul .title h2 {
-    color: #C7B002;
-    background: #00000000;
-}
-
-body .open-quest.tasks {
-    color: #f2f2f2; /* Sets the background color */
-}
-
-//Buttons
-body .window-app.forien-quest-preview .quest-body .quest-info .quest-col-right button {
-    flex: 0 0 1px;
-    font-size: 15px;
-    height: 20px;
-    line-height: 1;
-    white-space: nowrap;
-    background: $darkred;
-    color: white;
-}
-
-#forien-quest-log button, .window-app.forien-quest-preview button {
-    border-radius: 5px;
-    cursor: pointer;
-    height: 30px;
-    margin-bottom: 2px;
-    margin-right: 4px;
-    margin-top: 2px;
-    transition: border-color .3s ease,background .3s ease,box-shadow .3s ease;
-    background: $darkred;
-    color: white;
-}
-//Play
-#forien-quest-log .actions i.move.fas.fa-play:hover, .window-app.forien-quest-preview .actions i.move.fas.fa-play:hover {
-    color: green;
-}
-//Trashbin
-body #forien-quest-log .actions i.delete.fas.fa-trash:hover, body .window-app.forien-quest-preview .actions i.delete.fas.fa-trash:hover {
-    color: #cc5e5e;
-}
-
-body #forien-quest-log .actions i.fa-trash, body .window-app.forien-quest-preview .actions i.fa-trash {
-    color: #af0505;
-}
-//Back to deactive
-body #forien-quest-log .actions i.move.fas.fa-stop-circle:hover, body .window-app.forien-quest-preview .actions i.move.fas.fa-stop-circle:hover {
-    color: $edge;
-}
-
-//Make Available
-#forien-quest-log .actions i.move.fas.fa-clipboard:hover, .window-app.forien-quest-preview .actions i.move.fas.fa-clipboard:hover {
-    color: $edge;
-}
-
-body #forien-quest-log .quest-log .table ul .title p {
-    color: white;
-}
-//End Quest Log Styles
 
 .list-header.item-section[data-item-id="weapon"][data-item-type=""] {
```

Comment:
All styles from this point to the end of file live in the body / global scope (outside of .sr5). As far as I can see those should be system styles and be moved into .sr5
```CSS
.sr5 {
// all those rules 
}
```

bundle.scss already includes on large .sr5 which these can be added to.

Reply:
This is to avoid unwanted cross talk with other modules and Foundry. When those styles only affect windows that have the sr5 class, this can't happen. If some of those rules should affect more than system windows, then they should live in foundry.scss.

Reply:
Ok I won´t get that done until tomorrow, I will add some more fixes, modifier screen and Qualitys text issue.